WordPress/wporg-showcase-2022
# Turn off submit site link on that page.

Fixes: #80 

This PR hides the "Submit a site" link when viewing that page. I create a new template part to accomplish this. The con of this approach is that changes to the subnav styles will need to be made in 2 places now. However, I don't think this is a big risk and is less than likely.

**Alternative Approaches**
- I looked at targeting via CSS but because there is some mobile code that adds a dropdown button, we would get an empty menu. If we only hide it on desktop, it's a bit weird to have different content.
- I briefly considered adding some php/js logic but it seems like overkill.

## How To Test
- Visit `/submit-a-wordpress-site`, expect to have no "Submit a site" navigation link.
